### PR TITLE
[DO NOT MERGE] Top Level WAF Resource changes : Change from PR #5906 + some review comments

### PR DESCRIPTION
**Pull request by venk8**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by dsgouda** [comment]
@venkatsvpr please link the original PR here for context

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -532,6 +565,113 @@
...
+        }
+      }
+    },
+    "ManagedRuleSet": {
+      "type": "object",
+      "description": "Base class for all types of ManagedRuleSet.",
+      "x-ms-discriminator-value": "Unknown",
+      "properties": {
+        "ruleSetVersion": {
+          "description": "defines version of the rule set",
```
Comment (EvgenyAgafonchikov) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Please update to have first letter capitalized and dot in the end (for all the descriptions). Feel free to reuse any commits or create patch from [there](https://github.com/Azure/azure-rest-api-specs/pull/5965).
You could find more info on this [there](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-authoring-descriptions.md#basic-rules-for-public-docs-descriptions).

Currently NetworkValidation step of CI already checks if the first letter capitalized and later we are going to introduce more checks.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json [hunk 2]
```diff
@@ -288,13 +288,21 @@
           "description": "Describes  policySettings for policy",
```
Comment (EvgenyAgafonchikov) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: minor: double space after "Describes"

**Review by EvgenyAgafonchikov — CHANGES_REQUESTED** [review]
Added some comments, please take a look

**Comment by venkatsvpr** [comment]
[[Link] ](https://github.com/Azure/azure-rest-api-specs/pull/5906)to original PR for context

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json (hunk 1 above)
Comment (venkatsvpr) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Done

File: specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json (hunk 2 above)
Comment (venkatsvpr) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json]: Done

**Comment by MikhailTryakhov** [comment]
will be closed tomorrow when it'll be merged

**Comment by MikhailTryakhov** [comment]
closing due to NOT FOR MERGE

**Comment by venkatsvpr** [comment]
Will be adding some more changes to the PR.

**Pull request closed without merging**